.github/workflows/test.yaml

```diff
@@ -0,0 +1,81 @@
+name: KBase Handle Service 2 test
+
+on:
+  pull_request:
+    types:
+    - opened
+    - reopened
+    - synchronize
+    - ready_for_review
+  push:
+    # run workflow when merging to main or develop
+    branches:
+      - main
+      - master
+      - develop
+
+jobs:
+  handle_service_tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - python: '3.9.19'
+            mongo: 'mongodb-linux-x86_64-3.6.23'
+            wired_tiger: 'false'
+
+    steps:
+    - name: Check out GitHub repo
+      uses: actions/checkout@v4
+
+    - name: Setup Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: ${{matrix.python}}
+
+    - name: Install dependencies and set up test config
+      shell: bash
+      env:
+        AUTH_SERVICE_URL: https://ci.kbase.us/services/auth/api/legacy/KBase/Sessions/Login
+        AUTH_URL: https://ci.kbase.us/services/auth
+        BLOBSTORE_URL: https://ci.kbase.us/services/shock-api
+        ADMIN_TOKEN: ${{secrets.KBASE_CI_TOKEN}}
+      run: |
+
+        # install python libraries
+        python -m pip install --upgrade pip
+        pip install pipenv
+        pipenv sync --system
+
+        # set up mongo
+        cd ..
+        wget -q http://fastdl.mongodb.org/linux/${{matrix.mongo}}.tgz
+        tar xfz ${{matrix.mongo}}.tgz
+        export MONGOD=`pwd`/${{matrix.mongo}}/bin/mongod
+        cd -
+
+        # set up test config
+        cp -n test.cfg.example test.cfg
+        sed -i "s#^test.temp.dir =.*#test.temp.dir=temp_test_dir#" test.cfg
+        sed -i "s#^test.mongo.exe.*#test.mongo.exe=$MONGOD#" test.cfg
+        sed -i "s#^test.mongo.wired_tiger.*#test.mongo.wired_tiger=${{matrix.wired_tiger}}#" test.cfg
+
+        sed -i "s#^auth-service-url.*#auth-service-url=$AUTH_SERVICE_URL#" test.cfg
+        sed -i "s#^auth-url.*#auth-url=$AUTH_URL#" test.cfg
+        sed -i "s#^blobstore-url.*#blobstore-url=$BLOBSTORE_URL#" test.cfg
+        sed -i "s#^admin-token.*#admin-token=$ADMIN_TOKEN#" test.cfg
+        cat test.cfg
+
+    - name: Run tests
```

Comment:
I think is okay if it shows `admin-token=***`

Reply:
If the contents of the file are admin-token=*** how do the tests succeed?

Reply:
token is masked here. What are you asking?

Reply:
Are you saying the contents of the file are different from what cat displays?

Reply:
https://github.com/kbase/handle_service2/actions/runs/9022269670/job/24791398060?pr=107#step:4:110

this token is from `secrets`, therefore masked.

Reply:
Huh, ok, I've always just avoided displaying secrets manually, I didn't know GHA auto masked them. That said, now the tests are passing there's no real need to cat the file, and in general catting files with secrets in public places is a bad idea, so I still think we should delete it.

Plus it's misleading to certain people who shall not be named who might think the tests are failing because the token is set to `***`

Reply:
👍

Reply:
:+1: